AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/Model/Storage.smithy

```diff
@@ -10,15 +10,15 @@ namespace aws.cryptography.keyStore
 //#- ActiveHierarchicalSymmetricVersion [HierarchicalSymmetricVersion](#hierarchicalsymmetricversion)
 //#- HierarchicalSymmetricVersion [HierarchicalSymmetricVersion](#hierarchicalsymmetricversion)
 //#- ActiveHierarchicalSymmetricBeacon
-@documentation("Configures the kind of key that an encrypted blob represents.")
+@documentation("Describes the key that an encrypted blob represents.")
 union HierarchicalKeyType {
-  @documentation("The version the active branch key version. This version is used to encrypt messages")
+  @documentation("The version the active branch key type. This version is used to encrypt messages.")
```

Comment:
Issue: This sentence is unreadable.

"The x the y" is a poor construction.
Why not "The active branch key type. The active is used to encrypt messages." ?

Why include the term "version"?

Reply:
```suggestion
  @documentation("The version the active branch key. This version is used to encrypt messages.")
```